src/runtime.rs

```diff
@@ -307,6 +328,20 @@ impl Class {
 
     }
 
+    /// Checks whether this class conforms to the specified protocol.
+    pub fn conforms_to_protocol(&self, proto: &Protocol) -> bool {
```

Comment:
Bikeshedding: what are your thoughts on naming this `conforms_to` vs `conforms_to_protocol`?

I got thinking about this from how Objective-C puts argument names and types in its method names, but rust doesn't have to with its stronger type system (and also lack of argument labels). I haven't been able to totally figure out a convention, the closest I've found is `UdpSocket::send_to` rather than `UdpSocket::send_to_address`.

Anyways I think you could make an argument either way. Personally I think I slightly prefer the brevity of `conforms_to`, and it doesn't seem like it would lose meaning when reading code due to the surrounding context. Let me know what you think :)